en/04_Changelogs/5.0.0.md

```diff
@@ -494,6 +494,7 @@ This is a major release and contains many breaking API changes. Deprecation warn
 - The legacy file resolution strategy introduced in CMS 4.4.0 is no longer available. If you still use the legacy file resolution strategy, follow the [file migration instructions](https://docs.silverstripe.org/en/4/developer_guides/files/file_migration/) and then change your file resolution configuration to match the defaults in [the assets.yml file in silverstripe/installer](https://github.com/silverstripe/silverstripe-installer/blob/4/app/_config/assets.yml).
 - Removed the `HTMLValue` injection "shorthand", use the fully qualified [`HTMLValue`](api:SilverStripe\View\Parsers\HTMLValue) instead.
 - In `silverstripe/staticpublishqueue` the class [`SilverStripe\StaticPublishQueue\Dev\StaticPublisherState`](api:SilverStripe\StaticPublishQueue\Dev\StaticPublisherState) is no longer enabled by default and can be enabled via opt-in. There are opt-in instructions in the `README.md` of the module.
+- PDO database connector support has been fully removed
```

Comment:
This belongs under ORM changes - and should probably include instructions for what to do if you're using PDO currently (i.e. the correct ENV var value to set)

Reply:
Updated